--- population.f90	2017-06-11 22:41:29.406343605 -0400
+++ population_new.f90	2017-06-11 22:46:17.734466468 -0400
@@ -589,6 +589,7 @@
 real*8 atmdipx(ncenter),atmdipy(ncenter),atmdipz(ncenter),charge(ncenter)
 real*8 :: covr_becke(0:nelesupp) !covalent radii used for Becke population
 character selectyn,chgfilename*200
+character radfilename*800
 integer :: nbeckeiter=3
 
 if (chgtype==5) then !Select atomic radii for Becke population
